projects/plugins/jetpack/3rd-party/3rd-party.php

```diff
@@ -75,11 +57,8 @@ function load_3rd_party_compat_filters() {
 		require_once JETPACK__PLUGIN_DIR . '/3rd-party/debug-bar.php';
 	}
 
-	// CRM. Always included; it is used only when certain endpoints are hit.
-	require_once JETPACK__PLUGIN_DIR . '/3rd-party/class-jetpack-crm-data.php';
-
 	// Special case. Tools to be used to override module settings.
-	require_once JETPACK__PLUGIN_DIR . '/3rd-party/class-jetpack-modules-overrides.php';
+	\Jetpack_Modules_Overrides::instance();
```

Comment:
Probably no need to do this here. All it does is fill in the private `$instance` field, which could just as well happen if something else calls the method later.